.github/resign.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"fmt"
+	"github.com/mudler/luet/pkg/api/client"
+	"github.com/mudler/luet/pkg/api/core/types"
+	"github.com/mudler/luet/pkg/installer"
+	"io/ioutil"
+	"os"
+	"os/exec"
+)
+
+func getRepositoryPackages(repo string, ctx *types.Context) (searchResult client.SearchResult) {
+	tmpdir, err := ioutil.TempDir(os.TempDir(), "ci")
+	if err != nil {
+		panic(err)
+	}
+	defer os.RemoveAll(tmpdir)
+	referenceID := os.Getenv("REFERENCEID")
+	if referenceID == "" {
+		referenceID = "repository.yaml"
+	}
+	d := installer.NewSystemRepository(types.LuetRepository{
+		Name:        "cOS",
+		Type:        "docker",
+		Cached:      true,
+		Urls:        []string{repo},
+		ReferenceID: referenceID,
+	})
+	ctx.Config.GetSystem().Rootfs = "/"
```

Comment:
huh, actually if we bump the luet module now this is changed and much easier to work with

Reply:
I dont even know what that is used for, I blindly copied it from the build.go and it didnt gave me any issues so working as intended LOL